tests/integrationv2/providers.py

```diff
@@ -477,6 +480,16 @@ def supports_cipher(cls, cipher, with_curve=None):
 
         return True
 
+    def _client_major_ver(self):
+        result = subprocess.run(["openssl", "version"], shell=False, capture_output=True, text=True)
+
+        version_str = result.stdout.split(" ")
+        project = version_str[0]
+        version = version_str[1]
+        print(f"Project: {project} version: {version}")
+        assert project == "OpenSSL"
+        assert version.split('.')[0] == "1"
+
```

Comment:
According to what I understand, this section of the code makes sure that the test is being run against OpenSSL version 1.x.The test will only succeed if it is being performed against an OpenSSL 1.x version, which is what we are stating to be the case ?

Reply:
Right- I'm trying to prevent future me from wasting time troubleshooting openssl3 s_client issues until https://github.com/aws/s2n-tls/issues/3963 is resolved.

Reply:
Maybe return a bool and rename to `is_openssl_111`?

Reply:
But I'm trying to halt testing and tell the CI/user that this isn't supported...changed it to a `raise` and did the renaming.